src/react-medium-image-zoom.js

```diff
@@ -138,6 +140,7 @@ export default class ImageZoom extends Component {
       delete this.portalInstance
       delete this.portal
     }
+    this.props.onUnzoom()
```

Comment:
This seems to be getting called twice, meaning that this function is getting called twice, which I don't think it should and could give this new functionality some double responses (which is not good)